src/app.ts

```diff
@@ -100,11 +100,18 @@ export default class CeramicAnchorApp {
         break;
       }
       default: {
-        logger.err(`Unknown application mode ${mode}`);
+        logger.err(`Unknown application mode ${config.mode}`);
         process.exit(1);
       }
     }
-    logger.imp(`Ceramic Anchor Service initiated ${mode} mode`);
+    logger.imp(`Ceramic Anchor Service initiated ${config.mode} mode`);
+  }
+
+  private async _normalizeConfig(): Promise<void> {
```

Comment:
Can we roll in the `_patchConfigTypes` into this?

Otherwise code seems like it could get confusing as the config gets updated in multiple places many times

Reply:
Ooh good call.  Done